plugins/gui_filepicker.lua

```diff
@@ -1,143 +1,188 @@
 -- mod-version:3
 
+-- ** LINUX ONLY **
+-- KDE Dialog: Install kdialog package.
+-- GTK Zenity: Install zenity package (default).
+
 local core = require("core")
 local command = require("core.command")
 local common = require("core.common")
 local keymap = require("core.keymap")
+local config = require("core.config")
 local TreeView = require("plugins.treeview")
 
-local function run_zenity(options, callback)
-	local zen, proc_err = process.start({
-		"zenity",
-		table.unpack(options)
-	})
-	if not zen then
-		core.error("Unable to run zenity: %s", proc_err)
-		return
-	end
-
-	-- Run this in a coroutine, so we don't block the UI
-	core.add_thread(function()
-		local buffer = {}
-		repeat
-			local buf = zen:read_stdout()
-			if buf and #buf > 0 then
-				table.insert(buffer, buf)
-			end
-			coroutine.yield()
-		until not buf
-		local abs_path = table.concat(buffer) or ""
-
-		-- Remove final newline zenity adds
-		abs_path = string.match(abs_path, "^[^\n]+") or ""
-		if #abs_path == 0 then
-			return
-		end
-
-		callback(abs_path)
-	end)
+config.plugins.gui_filepicker = common.merge({
+    dialogUtility = "zenity",
```

Comment:
Let's use snake case:
```suggestion
    dialog_utility = "zenity",
```